### ISSUE TITLE
Meta-data issues

### ISSUE THREAD

**Issue by hvr** [bflyblue/bolt#1]
Welcome to Hackage!

I noticed your recent first package upload to Hackage as it was flagged by the matrix buildbot due to  packaging meta-data issues:

- The `description` ("Please see README.md") is not a proper description for your package (and Hackage will show a link to the README at the end of the description *if* the README exists; therefore that phrase is redundant), and hence this will cause poor experience for users (NB: your package won't be properly indexed by Hackage's package search feature and thus hurt its discoverability). See also [this posting](https://mail.haskell.org/pipermail/haskell-cafe/2016-November/125541.html) for more details.

- The build-dependencies have inaccurate versions bounds. Please be aware that accurate version bounds are mandated by the [PVP](http://pvp.haskell.org/), and adherence to the PVP is expected as part of the Hackage guidelines.  This has various knock off consequences, such as Hackage possibly not being able to generate Haddock documentation for your package, as well generally resulting in a poor experience for users since they'll run into the same compilation errors. Here's just one example: When an install-plan with  `network < 2.6` and `network-uri >= 2.6` is picked, you'll run into a compilation failure such as

```
In order, the following will be built (use -v for more details):
 - bolt-0.1.0.1 {bolt-0.1.0.1-inplace} (lib) (first run)
Preprocessing library bolt-0.1.0.1...

src/Database/Bolt/Transport/Socket.hs:27:18:
    Ambiguous module name ‘Network.URI’:
      it was found in multiple packages:
      network-2.5.0.0@HwJJxUkjiBFBQIlEtrjEY1 network-uri-2.6.1.0@4xX5aB9sRUAIM5zxcdBosg
```

- Moroever, In the case of `bolt`, not specifying `other-extensions: DerivingAnyClass` results in the cabal solver (and therefore the buildbot) picking a bad install-plan with GHC 7.8 (which didn't have this extension yet):

```
Configuring component lib from bolt-0.1.0.1
Preprocessing library bolt-0.1.0.1...

src/Data/PackStream.hs:1:14: Unsupported extension: DeriveAnyClass
```

If there is anything unclear of if you have any questions, please let me know!

**Comment by bflyblue** [comment]
Thank you! I'll address these issues shortly. 

-shaun

**Comment by hvr** [comment]
The 0.1.0.3 release looks good to me, thanks!

I've backported the meta-data fixes to 0.1.0.1 and 0.1.0.2 via meta-data editing (as otherwise the cabal solver would be able to backtrack to an earlier release), see

 - https://hackage.haskell.org/package/bolt-0.1.0.1/revisions/
 - https://hackage.haskell.org/package/bolt-0.1.0.2/revisions/

I have a minor nitpick though: from the 0.1.0.1 to the 0.1.0.2 release a new module was added (i.e. `Database.Bolt.Protocol.Ver1.Graph`). According to the http://pvp.haskell.org/ such an API addition would have required a minor version bump (i.e. 0.1.**1**.0 rather than 0.1.0.2)
